src/schema/sale/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -1,5 +1,49 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Sale type artworksConnection returns data from gravity 1`] = `
+Object {
+  "sale": Object {
+    "artworksConnection": Object {
+      "edges": Array [
+        Object {
+          "node": null,
```

Comment:
I don't think the snapshot is right, these are all `null`. I think you might need to stub the data for `saleArtworksLoader` (there's the `invert`, etc.).

probably something like: `{id: "sale_artwork_id", artwork: { id: "artwork_id", ... } }`